commands/liquidity_provision_submission.go

```diff
@@ -73,117 +71,6 @@ func checkLiquidityProvisionSubmission(cmd *commandspb.LiquidityProvisionSubmiss
 			errs.AddForProperty("liquidity_provision_submission.fee", ErrMustBePositive)
 		}
 	}
-
```

Comment:
Should  we have an explicit error returned if the users have been setting the shape? Probably not so useful, just to at least notify users that the shapes have been deprecated?

Reply:
well, for spot there is one, not at this level. In here we should add one on @karlem PR when deprecating the shapes, but as of this PR they're still valid for futures.

Reply:
hum, fair point.